changelogs/fragments/1209-lambda_event-add-support-for-function_response_types.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - lambda_event -  Added support to set FunctionResponseTypes when creating lambda event source mappings (https://github.com/ansible-collections/amazon.aws/pull/1209).
```

Comment:
```suggestion
  - lambda_event -  Added support to set ``function_response_types`` when creating lambda event source mappings (https://github.com/ansible-collections/amazon.aws/pull/1209).
```

Reply:
@GomathiselviS @alinabuzachis currently I don't think we have integration tests for `lambda_event`, would it be fine to add tests for this PR under integration tests for `lambda` https://github.com/ansible-collections/amazon.aws/tree/main/tests/integration/targets/lambda ?

Reply:
I would probably go with a new test suite for `lambda_event`, but it's up to you.

Reply:
I feel if it's okay, I'd like to get this PR merged first and open new PR for adding test suite?
@alinabuzachis @gravesm @GomathiselviS

Reply:
In general, we should really be adding tests along with the code. Tests can often highlight problems with a PR that aren't obvious by just reviewing. There's also the danger that even with the best intentions, the tests never get added later due to any number of reasons.

Reply:
Okay, will add test suite to this PR soon.